### PR TITLE
Python wheels

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

**Review by jcfr — APPROVED** [review]
:fireworks:

File: CMakeLists.txt [hunk 1]
```diff
@@ -4,6 +4,20 @@ cmake_minimum_required(VERSION 2.8.9)
 if(NOT ITK_SOURCE_DIR)
   find_package(ITK 4.10 REQUIRED)
   list(APPEND CMAKE_MODULE_PATH ${ITK_CMAKE_DIR})
+
+  if(SKBUILD)
```
Comment (fbudin69500) [inline CMakeLists.txt:8]: Should this eventually be added in a CMake macro in ITK?

Comment (thewtex) [reply CMakeLists.txt:8]: Yes, added here:

  http://review.source.kitware.com/#/c/22383/

Comment (fbudin69500) [reply CMakeLists.txt:8]: :+1:

**Pull request merged**